flutter/Graminsta/lib/post/mark_service.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:Graminsta/service/http_service.dart';
+import 'package:Graminsta/models/post.dart';
+import 'package:flutter/cupertino.dart';
+
+class MarkService {
+  static MarkService _instance;
+
+  factory MarkService() {
+    _instance ??= MarkService._();
+    return _instance;
+  }
+
+  MarkService._();
+
+  static Future<bool> addMark(Post post) async {
+    final response = await http.post("post/add-mark/",
+      body: {
+        "post_id": post.id.toString(),
```

Comment:
You can use `'${post.id}'` to get the string format instead of calling `.toString()` directly.